### ISSUE TITLE
User Input Malformed message

### ISSUE THREAD

**Issue by BobRathke** [moralmunky/Home-Assistant-Mail-And-Packages#379]
Thanks for your excellent work!! I have been running this package for several months and it has been terrific! After recently upgrading I get a "User Input Malformed" when trying to save the second page of the config. I want to add some shippers.

**Describe the bug**

**Environment (please complete the following information):**
 - OS: [e.g. HassOS/Raspbian/CentOS]
 HASS.IO
 - Type of device: [e.g. RPi3/NUC/Synology]
RPI3
 - Home Assistant version: [e.g. 0.105.5]
Home Assistant OS 5.12
- Hassio/Docker/Core? 
core-2021.2.3
 - Component version: [e.g. 0.1.2]
supervisor-2021.02.11
**Logs**
Paste your error logs here.

**Screenshots**
![image](https://user-images.githubusercontent.com/24194858/108905064-3f9ec200-75ed-11eb-99a2-40d36c4c9262.png)

**Comment by firstof9** [comment]
This error is due to the "Amazon Fwds" entry being empty.

If you put in `""` in that field, you'll be able to save.

**Comment by BobRathke** [comment]
Perfect Thanks!

I added in FedEx and confirmed I got the below email message  on a delivery. It doesn't appear in the sensor.
--

spacer.gif 	Hi, XXXXX. Your package from BASS PRO CATALOG-OB-PRIM ACCT is on its way.	spacer.gif
	
SCHEDULED DELIVERY
Thu, 02/25/2021
by 8:00pm
	
	
PICKED UP
SPRINGFIELD, MO
	
	
Not going to be home? Click an option below to take action.
	
Schedule Your Delivery*
Request an appointment delivery Tuesday - Saturday.
	
	* Some delivery options may not be available on the day of delivery.	
MANAGE DELIVERY
	
TRACKING NUMBER		952400998658 	
FROM		BASS PRO CATALOG-OB-PRIM ACCT
2300 E TURNER ST
SPRINGFIELD, MO, US, 65898
REFERENCE		851636100
SHIPPER REFERENCE		851636100
SHIP DATE		Tue 2/23/2021 01:09 PM
PACKAGING TYPE		Your Packaging
ORIGIN		SPRINGFIELD, MO, US, 65898
DESTINATION		ONTARIO, NY, US, 14519
SPECIAL HANDLING		Deliver Weekday
STANDARD TRANSIT		Thu, 02/25/2021 by 8:00pm
NUMBER OF PIECES		1
TOTAL SHIPMENT WEIGHT		2.00 LB
SERVICE TYPE		FedEx 2Day
